docs/docs/kubernetes/cli/compliance.md

```diff
@@ -29,173 +33,11 @@ trivy k8s cluster --compliance=nsa --report summary --format json
-| 5.1.6  | Ensure that Service Account Tokens are only mounted where necessary                                    |
-| 5.1.8  | Limit use of the Bind, Impersonate and Escalate permissions in the Kubernetes cluster                  |
-| 5.2.2  | Minimize the admission of privileged containers                                                        |
-| 5.2.3  | Minimize the admission of containers wishing to share the host process ID namespace                    |
-| 5.2.4  | Minimize the admission of containers wishing to share the host IPC namespace                           |
-| 5.2.5  | Minimize the admission of containers wishing to share the host network namespace                       |
-| 5.2.6  | Minimize the admission of containers with allowPrivilegeEscalation                                     |
-| 5.2.7  | Minimize the admission of root containers                                                              |
-| 5.2.8  | Minimize the admission of containers with the NET_RAW capability                                       |
-| 5.2.9  | Minimize the admission of containers with added capabilities                                           |
-| 5.2.10 | Minimize the admission of containers with capabilities assigned                                        |
-| 5.2.11 | Minimize the admission of containers with capabilities assigned                                        |
-| 5.2.12 | Minimize the admission of HostPath volumes                                                             |
-| 5.2.13 | Minimize the admission of containers which use HostPorts                                               |
-| 5.3.1  | Ensure that the CNI in use supports Network Policies (Manual)                                          |
-| 5.3.2  | Ensure that all Namespaces have Network Policies defined                                               |
-| 5.4.1  | Prefer using secrets as files over secrets as environment variables (Manual)                           |
-| 5.4.2  | Consider external secret storage (Manual)                                                              |
-| 5.5.1  | Configure Image Provenance using ImagePolicyWebhook admission controller (Manual)                      |
-| 5.7.1  | Create administrative boundaries between resources using namespaces (Manual)                           |
-| 5.7.2  | Ensure that the seccomp profile is set to docker/default in your pod definitions                       |
-| 5.7.3  | Apply Security Context to Your Pods and Containers                                                     |
-| 5.7.4  | The default namespace should not be used                                                               |
-```
-</details>
+| Compliance | Name for command | More info
+--- | --- | ---
+NSA, CISA Kubernetes Hardening Guidance v1.2 | `nsa` | [Link](https://media.defense.gov/2022/Aug/29/2003066362/-1/-1/0/CTR_KUBERNETES_HARDENING_GUIDANCE_1.2_20220829.PDF)
```

Comment:
We rename `nsa` to `k8s-nsa-1.2` or something similar, right?

Reply:
Chen told me our compliance report currently supports v1.0, not v1.2.
https://github.com/aquasecurity/trivy/pull/3349#discussion_r1058832132